### PR TITLE
add new changes

### PR DESCRIPTION
**Pull request by norona-marynicole**
add new changes

### CODE REVIEW

File: norona_marynicole/script.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+function checkFormValidity() {
+  let nameValue = nameInput.value.trim();
+  let commentValue = commentTextarea.value.trim();
+  commentButton.disabled = !(nameValue && commentValue);
+}
+
+function addComment() {
+  const nameInput = document.getElementById("name").value;
+  const commentInput = document.getElementById("gcomment").value;
+  if (nameInput.trim() && commentInput.trim()) {
```
Comment (Valerio-SirLance) [inline norona_marynicole/script.js]: use guard clause

**Review by Valerio-SirLance — CHANGES_REQUESTED** [review]
guard clause is usable, use it as said js guidelines

File: norona_marynicole/script.js (hunk 1 above)
Comment (norona-marynicole) [reply norona_marynicole/script.js]: Done revising po. Thank you po.

**Review by Valerio-SirLance — APPROVED** [review]

**Comment by Valerio-SirLance** [comment]
so far the code files are good to go, but there seems to be a problem in your Netlify deployment.

**Pull request merged**